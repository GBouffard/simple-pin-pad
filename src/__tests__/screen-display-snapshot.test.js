import React from 'react';
import ScreenDisplay from '../components/atoms/screen-display';
import renderer from 'react-test-renderer';

describe('ScreenDisplay - ', () => {
  const component = renderer.create(
    <ScreenDisplay
      display='this is a snapshot test' />
  );

  const tree = component.toJSON();

  it('renders correctly', () => {
    expect(tree).toMatchSnapshot();
  });
});