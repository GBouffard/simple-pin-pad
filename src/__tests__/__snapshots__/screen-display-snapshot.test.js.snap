// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ScreenDisplay -  renders correctly 1`] = `
<div
  className="App__screen-display"
>
  this is a snapshot test
</div>
`;
